azure-pipelines.yml

```diff
@@ -122,7 +122,7 @@ jobs:
   - task: UsePythonVersion@0
     displayName: 'Use Python 3'
     inputs:
-      versionSpec: 3.x
+      versionSpec: 3.8
```

Comment:
You could revert this file as this PR https://github.com/Azure/azure-cli/pull/15804 has fixed the build issue

Reply:
Then merge from `release` to get a clear branch history.